### PR TITLE
Add sections on setting cluster and project node selectors

### PR DESCRIPTION
**Pull request by mburke5678**
https://jira.coreos.com/browse/POD-94
https://jira.coreos.com/browse/POD-119

### CODE REVIEW

**Comment by mburke5678** [comment]
@ravisantoshgudimetla PTAL

File: modules/nodes-scheduler-node-selectors-cluster.adoc [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+spec:
+  nodeSelector:
+    region: west
+----
+
+If the project where you are creating the pod has a project node selector, that selector takes preference over a cluster node selector.
+
+.Procedure
+
+To add a default node selector to a project:
```
Comment (ravisantoshgudimetla) [inline modules/nodes-scheduler-node-selectors-cluster.adoc]: This is across cluster not per project

**Comment by mburke5678** [comment]
@sunilcio Is this an issue you can review?

**Comment by sunilcio** [comment]
@mburke5678 let me check now.

**Comment by mburke5678** [comment]
@sunilcio Any thoughts on this? I know we are all busy. But I would like to get this merged in. It seems like an important feature that we missed.

**Comment by sunilcio** [comment]
@mburke5678 I followed the steps in doc and it works correctly. LGTM.
I would recommend to also add statement from https://bugzilla.redhat.com/show_bug.cgi?id=1699460#c12 to something like below in Step 1 (Creating default cluster-wide node selectors)
`After making the change to the defaultNodeSelector in the cluster Scheduler config resource, wait for the kube-apiserver pods to redeploy.  This can take several minutes. Until the kube-apiserver pods redeploy, the defaultNodeSelector will not have effect.`

**Comment by mburke5678** [comment]
@openshift/team-documentation PTAL

**Pull request merged**